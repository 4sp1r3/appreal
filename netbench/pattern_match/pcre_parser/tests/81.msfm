42
43
0
26|0|27|0
27|0|28|1
28|1|29|0
29|0|30|1
30|2|31|0
31|0|32|1
32|3|33|0
33|0|34|1
34|4|35|0
35|0|36|1
36|5|37|0
37|0|38|1
38|6|39|0
39|0|40|1
40|7|41|0
41|0|13|1
14|8|15|0
15|0|16|1
16|9|17|0
17|0|18|1
18|10|19|0
19|0|20|1
20|11|21|0
21|0|22|1
22|12|23|0
23|0|24|1
24|7|25|0
25|0|12|1
2|13|3|0
3|0|4|1
4|14|5|0
5|0|6|1
6|15|7|0
7|0|8|1
8|16|9|0
9|0|10|1
10|7|11|0
11|0|12|1
1|0|14|1
1|0|2|1
12|0|13|1
0|0|26|1
0|0|1|1
######################################################################################
1
13,
######################################################################################
17
0:0x24|
1:0x6d|
2:0x79|
3:0x6e|
4:0x69|
5:0x63|
6:0x6b|
7:0x20|
8:0x24|
9:0x6c|
10:0x6f|
11:0x63|
12:0x6b|
13:0x24|
14:0x6b|
15:0x65|
16:0x79|
