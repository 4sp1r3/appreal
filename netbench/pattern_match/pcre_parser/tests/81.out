lex	: modif_front: SLASH
yacc	: pcre_delim <<< SLASH
yacc	: modif_front <<< pcre_delim
lex	: BOL
yacc	: bol <<< BOL
lex	: LPAR
yacc	: startCapturing <<< LPAR
lex	: ASCII $
yacc	: element <<< ASCII $
yacc	: unit <<< element
lex	: ASCII m
yacc	: ext_unit <<< unit
yacc	: element <<< ASCII m
yacc	: unit <<< element
lex	: ASCII y
yacc	: ext_unit <<< unit
yacc	: element <<< ASCII y
yacc	: unit <<< element
lex	: ASCII n
yacc	: ext_unit <<< unit
yacc	: element <<< ASCII n
yacc	: unit <<< element
lex	: ASCII i
yacc	: ext_unit <<< unit
yacc	: element <<< ASCII i
yacc	: unit <<< element
lex	: ASCII c
yacc	: ext_unit <<< unit
yacc	: element <<< ASCII c
yacc	: unit <<< element
lex	: ASCII k
yacc	: ext_unit <<< unit
yacc	: element <<< ASCII k
yacc	: unit <<< element
lex	: ASCII SPACE 
yacc	: ext_unit <<< unit
yacc	: element <<< SPACE
yacc	: unit <<< element
lex	: OR
yacc	: ext_unit <<< unit
yacc	: exp <<< ext_unit
yacc	: exp <<< ext_unit-exp
yacc	: exp <<< ext_unit-exp
yacc	: exp <<< ext_unit-exp
yacc	: exp <<< ext_unit-exp
yacc	: exp <<< ext_unit-exp
yacc	: exp <<< ext_unit-exp
yacc	: exp <<< ext_unit-exp
yacc	: ext_exp <<< exp
yacc	: rv <<< ext_exp
yacc	: or <<< OR
lex	: ASCII $
yacc	: element <<< ASCII $
yacc	: unit <<< element
lex	: ASCII l
yacc	: ext_unit <<< unit
yacc	: element <<< ASCII l
yacc	: unit <<< element
lex	: ASCII o
yacc	: ext_unit <<< unit
yacc	: element <<< ASCII o
yacc	: unit <<< element
lex	: ASCII c
yacc	: ext_unit <<< unit
yacc	: element <<< ASCII c
yacc	: unit <<< element
lex	: ASCII k
yacc	: ext_unit <<< unit
yacc	: element <<< ASCII k
yacc	: unit <<< element
lex	: ASCII SPACE 
yacc	: ext_unit <<< unit
yacc	: element <<< SPACE
yacc	: unit <<< element
lex	: OR
yacc	: ext_unit <<< unit
yacc	: exp <<< ext_unit
yacc	: exp <<< ext_unit-exp
yacc	: exp <<< ext_unit-exp
yacc	: exp <<< ext_unit-exp
yacc	: exp <<< ext_unit-exp
yacc	: exp <<< ext_unit-exp
yacc	: ext_exp <<< exp
yacc	: rv <<< ext_exp
yacc	: or <<< OR
lex	: ASCII $
yacc	: element <<< ASCII $
yacc	: unit <<< element
lex	: ASCII k
yacc	: ext_unit <<< unit
yacc	: element <<< ASCII k
yacc	: unit <<< element
lex	: ASCII e
yacc	: ext_unit <<< unit
yacc	: element <<< ASCII e
yacc	: unit <<< element
lex	: ASCII y
yacc	: ext_unit <<< unit
yacc	: element <<< ASCII y
yacc	: unit <<< element
lex	: ASCII SPACE 
yacc	: ext_unit <<< unit
yacc	: element <<< SPACE
yacc	: unit <<< element
lex	: RPAR
yacc	: ext_unit <<< unit
yacc	: exp <<< ext_unit
yacc	: exp <<< ext_unit-exp
yacc	: exp <<< ext_unit-exp
yacc	: exp <<< ext_unit-exp
yacc	: exp <<< ext_unit-exp
yacc	: ext_exp <<< exp
yacc	: rv <<< ext_exp
yacc	: rv <<< rv-or|rv
yacc	: rv <<< rv-or|rv
yacc	: endCapturing <<< RPAR
yacc	: capturing <<< startCapturing rv endCapturing
yacc	: unit <<< capturing
lex	: SLASH
yacc	: ext_unit <<< unit
yacc	: exp <<< ext_unit
yacc	: ext_exp <<< exp
yacc	: rv <<< ext_exp
yacc	: inslash <<< bol-rv
yacc	: pattern <<< inslash
yacc	: pcre_delim <<< SLASH
yacc	: modif_rear <<< pcre_delim
yacc	: pcre <<< modif_front pattern modif_rear
